### DISCUSSION TITLE
Vehicles Physic

### DISCUSSION THREAD

**Discussion by Piotrek94K** [godotengine/godot-proposals#5630] [Scripting]
I don't know how to turn it into a code (so far I'm only modifying one game and trying to create my own) but maybe someone can and maybe we will have better vehicle physics in Godot. Thanks for your time. Constructive criticism is welcome.

As we know the physics of Godot is not the best (at least that's the opinion)

This or a similar code would make it easier and more realistic to create a game with vehicles

#player start spawn = x at speed = 0 km/h
if user clicked on the variable="up arrow" then
at every second with each second held
accelerate +1 km/h and move the vehicle:
if the speed is x km/h move the player
forward about 0,27 m/s * x km/h
about every second.

This can improve Godot physics and make game development easier ( I think)

**Comment by Zireael07** [comment]
This is neither helpful nor funny. Spawning something with a speed of zero and accelerating 1 km/s isn't the problem and takes literally a couple lines of GDScript. Physics is not just accelerating the player, it's massively more complex.
